docs/internals/data-structures.rst

```diff
@@ -738,7 +738,7 @@ b) with ``create --chunker-params 19,23,21,4095`` (default):
 
   mem_usage  =  0.31GiB
 
-.. note:: There is also the ``--no-files-cache`` option to switch off the files cache.
+.. note:: There is also the ``--cache-mode=disabled`` option to disable the files cache.
```

Comment:
hmm, use `--files-cache` for this option?

is more specific about which cache is meant.